lib/fliterbarItems.tsx

```diff
@@ -0,0 +1,15 @@
+export const FLITERBAR_ITEMS = [
```

Comment:
@TingChiehLin I think you didn't understand what I meant here. This array right now is hardcoded. If the list of movies changes, this array might be outdated. The goal is for this array to be dynamic, based on the top100movies. For example:

```js
import movies from "./top_100_movies.json";

function getAllFiltersFromMovies(movies) {
   // your solution here... this is like another leetcode exercise ;)
}

const genres = getGenresFromMovies(movies) 
```

Reply:
It's great now, good to go!

A minor improvement would be to move

```
  const genres = getAllFliterGenres();
```

to this file and export/import it directly in the react component

```
import { genres } from "../../lib/fliterbarItems";
```